### PR TITLE
README: added installation section

### PR DESCRIPTION
**Pull request by dmitrykurmanov**

### CODE REVIEW

**Comment by dmitrykurmanov** [comment]
it is helpful because of github repository name may differ from the npm package name

**Comment by jmeas** [comment]
The examples do show this, but I'm 👍 to adding this as well.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ For webpack, add a [resolve.alias](http://webpack.github.io/docs/configuration.h
 
 Browser support: IE8+ and everything else.
 
+## Installation
+
+```
+npm i xhr
```
Comment (jmeas) [inline README.md]: Change the `i` to `install`? I use the shorthand myself, but in my experience more developers are familiar with the long name than the shorthand. I don't think we want anyone to think, "what will running this command do?" – it'd be better to appeal to a wider audience for the install imo.

Comment (dmitrykurmanov) [inline README.md]: yes, why not, you are right

**Pull request merged**